docs/notebooks/flax_basics.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<p>Line 11: Why not just initialize W in the right shape instead of having a transpose?</p><p>Line 16: I don't like how everything is crowded. Good to add more whitespace like you would everywhere around commas and other operators. (maybe this should be two lines)</p><p>Line 16: I don't know if it's good to use vmap here. Readers are probably confused about it still (if they even understand it at all). I know the original JAX intro covered it but you can't assume readers read in the correct order. WDYT? You could use <code>jnp.dot</code>  just fine without vmap, it all still works (because then dot becomes a matrix product instead)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>L11: indeed, this comes from the fact that I always think about column vectors more than row so this seemed more natural to me, but there is no reason to do so, I'll change it.</p><p> </p><p>L16: adding space indeed, and I somehow just thought "I want to apply that to a batch" and used vmap but that's definitely an overkill. <code>jnp.dot</code>will  do the trick just fine.</p>